### PR TITLE
CNV-6480 Add note on WFFC limitations to virt documentation

### PR DESCRIPTION
**Pull request by bgaydosrh**
This PR addresses https://issues.redhat.com/browse/CNV-6480 

Please label [enterprise-4](https://issues.redhat.com/browse/enterprise-4).7, [enterprise-4](https://issues.redhat.com/browse/enterprise-4).6, [enterprise-4](https://issues.redhat.com/browse/enterprise-4).5

See Netify build for preview. Navigate to **Virtual Machines > Virtual Disks > Configuring local storage for virtual machines > Creating a storage class** and view the NOTE.

Re-tagging @awels for code review; copying @aglitke 
Kevin Goldblatt has signed off QE review in Jira.

Tagging for peer review in CNV-docs in Slack.

### CODE REVIEW

**Review by sjhala-ccs — COMMENTED** [review]
Looks good, just a few nits. Thanks!

File: modules/virt-creating-storage-class.adoc [hunk 1]
```diff
@@ -41,6 +41,13 @@ binding occur. Specify `WaitForFirstConsumer` to delay the binding and provision
 of a PV until after a pod that uses the persistent volume claim (PVC)
 is created. This ensures that the PV meets the pod's scheduling requirements.
 
+[NOTE]
+====
+Virtual Machines use datavolumes that are based on local PVs. Local PVs are bound to specific nodes. While the disk image is prepared for consumption by the virtual machine, it is possible that the virtual machine cannot be scheduled to the node where the local storage PV was previously pinned.
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply modules/virt-creating-storage-class.adoc:46]: Fixed.

File: modules/virt-creating-storage-class.adoc [hunk 2]
```diff
@@ -41,6 +41,13 @@ binding occur. Specify `WaitForFirstConsumer` to delay the binding and provision
...
+
+To solve this problem, use the Kubernetes pod scheduler to bind the PVC to a PV on the correct node. By using `StorageClass` with `volumeBindingMode` set to `WaitForFirstConsumer`, the binding and provisioning of the PV is delayed until a Pod using the PVC is created.
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply modules/virt-creating-storage-class.adoc:48]: Good point. And I believe the object is what I am referencing. I also rephrased that last sentence. I think the verb should come first before the "using" clause.

**Pull request merged**